src/api/api.ts

```diff
@@ -202,7 +202,7 @@ const installedRequest = async (
     }
     
     if (isResponseError(response)) {
-        throw new Error(`Request failed: [${response.status}] ${await response.text()}`);
+        throw new Error(JSON.stringify({status:response.status,message:await response.text()}));
```

Comment:
I wonder is [pino](https://github.com/pinojs/pino) supports browser logging - think it does... so might be work considering to standardise our technical logging -- what do you think?

Reply:
Actually, we'd still want to bubble this to the error boundary -- so think pino is redundant... but could be useful if we're showing response debug in the browser

Reply:
I 100% agree, it's the one I use and I have a good setup for it in my personal typescript template